Haskell-Generate/GenPrimitive.hs

```diff
@@ -362,8 +362,10 @@ rawBenchmark "FullSubtract16" = rawBenchmark "FullSubtract32"
 rawBenchmark "FullSubtract64" = rawBenchmark "FullSubtract32"
 rawBenchmark "Multiply8" = rawBenchmark "Multiply32"
 rawBenchmark "Multiply16" = rawBenchmark "Multiply32"
+rawBenchmark "Multiply64" = rawBenchmark "Multiply32"
 rawBenchmark "FullMultiply8" = rawBenchmark "FullMultiply32"
 rawBenchmark "FullMultiply16" = rawBenchmark "FullMultiply32"
+rawBenchmark "FullMultiply64" = rawBenchmark "FullMultiply32"
```

Comment:
In 5cf9279e61e69f75103ca354ccbdb8c7c62eabf1:

Both of these added lines have 32s at the end which should be 64s.

Found by my local CI running GenPrimitive and confirming consistency between the committed primitiveEnumJet.inc and the one produced by running GenPrimitive.hs :).

Reply:
This block of code are stub benchmark timings that need to be filled in later (see the `TODO` above). So we are just reusing existing timings as placeholder.

Reply:
That's fair enough, but is there some way we can get the output of `GenPrimitive` to match the match the committed file?

The exact diff is
```
57a58
> FULL_MULTIPLY_64,
121a123
> MULTIPLY_64,
```
wherre the new lines are present in the committed file but not the generated one.

Reply:
Oh I see.  Yes something seems to have been lost in my rebasing.

Reply:
Should be fixed now.